repository/src/main/java/com/michaldrabik/repository/images/MovieImagesProvider.kt

```diff
@@ -118,13 +120,15 @@ class MovieImagesProvider @Inject constructor(
       }
     }
 
-  private fun findBestImage(images: List<TmdbImage>, type: ImageType) =
-    images
-      .filter { if (type == POSTER) it.isEnglish() else it.isPlain() }
```

Comment:
I actually want to to always use plain images for not-posters (wider fanarts) whenever available because of the UI design. The non-plain fanarts usually contain big titles etc. that don't work to well IMO.

For posters the priority can be language, then english and then plain.

Reply:
Thank you! From this comment I now understand that it is those plain (null) images that I didn't quite understand!

I change the order of preference in case it is not a poster!